src/Reader/readpar.f90

```diff
@@ -3301,6 +3301,25 @@ SUBROUTINE readpar_output(EQN,DISC,IO,CalledFromStructCode)
            IO%RotationMask(1:4) = .TRUE.
          ENDIF
 
+      ALLOCATE(IO%OutputRegionBounds(6),STAT=allocStat )                                      !
+       IF (allocStat .NE. 0) THEN                                                       !
+         logError(*) 'could not allocate IO%OutputRegionBounds in readpar!'!
+         STOP                                                                           !
+       END IF
+      IO%OutputRegionBounds(1:6) = OutputRegionBounds(1:6)
+      IF (OutputRegionBounds(2)-OutputRegionBounds(1) <= 0.0) THEN
+        logError(*) 'Please make sure the x bounds are correct'
```

Comment:
This might be a problem. First, OutputRegionBounds should have a default value (probably 0, 0, 0, 0, 0, 0). Second, the default value should correspond to "select all cells".